fleet/lib/cnpg-cluster/overlays/yagan/cnpg-loadBalancer-lhn.yaml

```diff
@@ -1,13 +1,12 @@
----
 apiVersion: v1
 kind: Service
 metadata:
-  annotations:
-    metallb.universe.tf/address-pool: lhn
+  name: cnpg-loadbalancer-lhn
   labels:
     cnpg.io/cluster: cnpg-cluster
-  name: cnpg-loadbalancer-lhn
-  namespace: cloudnativepg
+  annotations:
+    metallb.universe.tf/address-pool: lhn
```

Comment:
The pool doesn't need to be set when a specific IP is requested.

Reply:
no problem, i was just been extra specific lol, will remove it